source/Nabla/COpenGLDriver.h

```diff
@@ -221,24 +218,10 @@ class COpenGLDriver final : public CNullDriver, public COpenGLExtensionHandler
 	public:
         struct SAuxContext;
 
-		#ifdef _NBL_COMPILE_WITH_WINDOWS_DEVICE_
-		COpenGLDriver(const SIrrlichtCreationParameters& params, io::IFileSystem* io, CIrrDeviceWin32* device, const asset::IGLSLCompiler* glslcomp);
+		COpenGLDriver(const SIrrlichtCreationParameters& params, io::IFileSystem* io, asset::IAssetManager* assmgr, const asset::IGLSLCompiler* glslcomp);
 		//! inits the windows specific parts of the open gl driver
-		bool initDriver(CIrrDeviceWin32* device, EGLDisplay eglDisplay);
+		bool initDriver(CIrrDeviceStub* device);

```

Comment:
while you're at it... is it possible to rename the driver into the IAPIConnection (or whatever we call the logical instance?)

Reply:
thats what will be done, however its not as simple as rename obvs

Reply:
and the name is LogicalDevice btw,
APIConnection is on the level of VkInstance